testUtils/assertUtils.ts

```diff
@@ -0,0 +1,23 @@
+import { MatcherFunction, Nullish } from '@testing-library/react';
+
+type Query = (matcher: MatcherFunction) => HTMLElement;
+
+/**
+ * Decorate an existing query to match text across multiple
+ * elements.
+ * @param query
+ */
+export function withMarkup(query: Query) {
```

Comment:
This is a helper to find a text across multiple elements (especially useful for flash messages, where we use the resource names inside `<code />` blocks)